projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -82,6 +82,28 @@ frontend-data-pipelines-release:
       - projects/frontend/data-pipelines/gui/package-lock.json
     expire_in: 1 week
 
+frontend_publish_ui_image:
+  image: docker:19.03.15
```

Comment:
This is a very old version of the docker image. 
Can you use the newer one that we use in the control-plane gitlab files. 
Can you also extract all this docker config like we do in control-plane and extend it anywhere it is needed.